client/App.tsx

```diff
@@ -37,14 +41,15 @@ const App = (): ReactElement => {
   const handleTestClick = function (): void {
     console.table(tracks);
     console.log(genres);
+    fetchTracks();
```

Comment:
Obviously right now we are just using this button to console.log some data and to be able to test some functionality. It won't be part of the actual app. But I just wanted to point out that by running the `console.table(tracks)` before the `fetchTracks()` we will likely see what was saved to the `tracks` variable just before we set them to something new.